enemyAI.js

```
@@ -192,7 +192,7 @@ class EnemyAI {
             // Boss intro animation
             this.state = 'intro';
             this.bossIntroComplete = true;
-            if (typeof createBossTitle !== 'undefined') createBossTitle("Corrupted Sproutling");
+            if (typeof createBossTitle !== 'undefined') createBossTitle('Corrupted Sproutling');
```

ESLint

[error] 'createBossTitle' is not defined.

(no-undef)